wire-ios-data-model/Source/ManagedObjectContext/NSManagedObjectContext+NotificationContext.swift

```diff
@@ -25,6 +25,7 @@ extension NSPersistentStoreCoordinator: NotificationContext {}
 
 public extension NSManagedObjectContext {
 
+    /// Prefer `UserSession.notificationContext` over this property.
```

Comment:
What about declaring it deprecated?

Reply:
IMO it's not a good idea, because the property can not be simply replaced in many places. it will produce a lot of warnings (hundreds?). see `ZMUserSession.shared`, that pollutes the warnings, but can not be fixed easily.